### PR TITLE
uint: release 0.9.2

### PR DESCRIPTION
**Pull request by ordian**
Releasing as patch because of https://github.com/paritytech/parity-common/issues/577#issuecomment-893301299.

### CODE REVIEW

File: uint/CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,10 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+
+## [0.9.2] - 2022-01-28
 - Migrated to 2021 edition, enforcing MSRV of `1.56.1`. [#601](https://github.com/paritytech/parity-common/pull/601)
```
Comment (niklasad1) [inline uint/CHANGELOG.md:10]: shouldn't the MSRV bump be regarded as a breaking change?

Comment (ordian) [reply uint/CHANGELOG.md:10]: see my comment above

**Review by niklasad1 — APPROVED** [review]

File: uint/CHANGELOG.md (hunk 1 above)
Comment (niklasad1) [reply uint/CHANGELOG.md:10]: fair enough, got it

**Comment by ordian** [comment]
published cc @dalance @uint

**Pull request merged**